osmaxx/excerptexport/templates/excerptexport/email/all_exports_of_extraction_order_done_body.txt

```diff
@@ -1,10 +1,14 @@
-{% load navigation %}
-The extraction order #{{ extraction_order.id }} "{{ extraction_order.excerpt_name }}" has been processed.
-{% if successful_exports.count > 0 %}Results available for download:
-{% for export in successful_exports %}- {{ export.get_file_format_display }}{% if export.output_file.file %} ({{ export.output_file.file.url | siteabsoluteurl:request }}){% endif %}
+{% load navigation %}This is an automated email from the OSMaxx service.
```

Comment:
> Drop the "The." [...] It's cleaner

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&mdash;[The Social Network](http://www.imdb.com/title/tt1285016/quotes?item=qt1307232) (2010)

If you feel we need to distinguish between
* OSMaxx, the software
* the "OSMaxx project" (as contracted by the EDA)
* the OSMaxx instance run by Geometa Lab HSR incl. the service(s) this instance provides or that Geometa Lab provides in context of that instance

then I'd suggest the following nomenclature:
<dl><dt>OSMaxx</dt><dd>(without a "the")</dd><dd>the software</dd><dt>osmaxx.hsr.ch</dt><dd>(without a "the", without "service")</dd><dd>the instance run on https://osmaxx.hsr.ch and the service(s) provided by it/in conjunction with it</dd><dt>Auftrag „Bereitstellen von OpenStreetMap-Daten“</dt><dd>the project as contracted</dd></dl>

Of course, when referring to ourselves in our roles as the maintainers of osmaxx.hsr.ch, "the osmaxx.hsr.ch team" (here with a "the") would be fine.